stock_auto_move/__manifest__.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2014 NDP Systèmes (<http://www.ndp-systemes.fr>).
```

Comment:
Please use the [short header](https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3) instead